src/nc_request.c

```diff
@@ -112,6 +114,11 @@ req_put(struct msg *msg)
         msg->peer = NULL;
         pmsg->peer = NULL;
         rsp_put(pmsg);
+        /* only record the server pool message */
+        if (conn->client && !conn->proxy) {
+            req_time = nc_msec_now()-msg->start_ts/1000;
+            stats_pool_record_latency(conn_to_ctx(conn), conn->owner, req_time);
```

Comment:
memcache get command will been record twice in pool latency , please fix it before merging

Reply:
should not record fragment msg as pool latency，add check for fragment ?
            if (msg->frag_id == 0 || msg->frag_owner == msg) {

Reply:
Fixed